index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css" media="screen" charset="utf-8">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <title>Responsive Login Page</title>
+  </head>
+  <body>
+
+    <section>
+      <h1>Login</h1>
+      <form class="" method="post">
+        <label for="email">Email</label>
+        <br>
+        <input type="email" name="name" id="email">
```

Comment:
Instead of doing JS validation, you can just do HTML validation here (i.e. `required`)